### PR TITLE
fix: norwegian translations

### PR DESCRIPTION
**Pull request by tersor**
# Description
Updated translations for norwegian bokmål and added translations for norwegian nynorsk.

### CODE REVIEW

File: lib/generators/avo/templates/locales/avo.nb.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-nb-NO:
```
Comment (adrianthedev) [inline lib/generators/avo/templates/locales/avo.nb.yml:1]: Is this the right way of having the translations in Norway `nb` vs. `nb-NO`? I'm not very familiar with localization.

**Review by adrianthedev — APPROVED** [review]
Thank you for your contribution!

File: lib/generators/avo/templates/locales/avo.nb.yml (hunk 1 above)
Comment (tersor) [reply lib/generators/avo/templates/locales/avo.nb.yml:1]: Yes it is. [Example reference](https://github.com/svenfuchs/rails-i18n/commit/196ea80372e08941f9e9e8899e3af7b0b86e703c).

Comment (adrianthedev) [reply lib/generators/avo/templates/locales/avo.nb.yml:1]: Thank you @tersor! I appreciate the help.

Could we, in the future when we add new strings, ping you to validate, and help with Norwegian translations?

Comment (tersor) [reply lib/generators/avo/templates/locales/avo.nb.yml:1]: Jep, sure!

**Pull request merged**